contract/src/start-postalSvc.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file core eval script* to start the postalSvc contract.
+ *
+ * * see test-gimix-proposal.js to make a script from this file.
+ *
+ * The `permit` export specifies the corresponding permit.
+ */
+// @ts-check
+
+import { E } from '@endo/far';
+import { fixHub } from './fixHub.js';
+
+const trace = (...args) => console.log('start-postalSvc', ...args);
```

Comment:
Can you make this consistent with the version in `internal`, which defaults to disabled?

Reply:
it's only used once. I think I'll just use `console.log`.